website/src/app/page.tsx

```diff
@@ -174,51 +179,36 @@ export default function Page() {
           </div>
           <div>
             <h4 className="mb-8 text-lg font-semibold tracking-tight text-primary-450 ">
-              FAST AND LIGHTWEIGHT
+              RELIABLE ACCESS
             </h4>
-            <h3 className="text-2xl font-semibold tracking-tight text-neutral-900">
-              High throughput and low latency. <br />
-              Up to 4-6x faster than OpenVPN.
-            </h3>
             <p className="text-xl text-neutral-800 my-4">
-              Increase productivity and decrease connection issues for your
-              remote team. Firezone uses kernel WireGuard® to be efficient,
-              reliable, and performant in any environment.
+              Firezone is fast and dependable so your team is always connected
+              to the resources they need most. It works on all major platforms
+              and stays connected even when switching networks.
```

Comment:
Is it possible that switching networks could be misunderstood (like switching Gateways, internal networks, or something like that)? I wrote out mobile / WiFi because it's a differentiator for WireGuard and one of the big reliability / quality of life issues that people kept bringing up for Tailscale. Would something like "...stays connected even when switching WiFi networks." be appropriate here?

Reply:
Ah I see, yeah agreed, I'll update this copy.